src/devtools/gendbg.sh

```diff
@@ -36,9 +36,9 @@ if [ "$OUTFILEDIR" != "$INFILEDIR" ]; then
 	OUTFILE=${OUTFILEDIR}/${OUTFILE}
 fi
 
-pushd "$INFILEDIR"	
+pushd "$INFILEDIR" > /dev/null
```

Comment:
Why redir?

Reply:
push & popd print the directory stack whenever they're invoked, this just silences them

Reply:
Why do they need to be silenced?

Reply:
It's not really important, but without this, the build will print the following whenever a shared lib is copied:
```
---- COPYING TO ../../game/bin/engine.so [release] ----
/src/game/bin /src/src/engine
/src/src/engine
```
It's not particularly important, so I'll revert it if you want.

Reply:
Reverted